frontend/src/components/shared/RepoHeader.vue

```diff
@@ -141,11 +141,13 @@
     >
       <a
         class="md:ml-0 hover:text-brand-700 text-gray-500 font-normal"
+        v-if="repoType !== 'endpoint'"
         :href="ownerUrl"
       >
         {{ path?.split('/')[0] }}
       </a>
-      <div>/</div>
+      <div v-if="repoType !== 'endpoint'"
```

Comment:
is it a empty div?

Reply:
`<div  v-if="repoType !== 'endpoint'>/</div>` It's not empty